### DISCUSSION TITLE
Examples running on different version than latest release?

### DISCUSSION THREAD

**Discussion by richardrevesz** [google/model-viewer#2089] [Q&A] [answered]
Hi,
I am playing around with swap textures, and the example works fine both mobile and desktop, while on my web page it throws errors on mobile and the texture swap is not happening.
I can see my page imports the latest release (1.5.0 from unpkg.com), while on the examples page it is not clear which version is running.
The model-viewer.js starts differently and has about 6500 lines more in the latest release than it has on the examples page.

I get these errors on Samsung Galaxy S8, Android 9 (Pie), Chrome 89.0.4389.72:
- model-viewer.js:16306 THREE.WebGLRenderer: OES_texture_float extension not supported.
- model-viewer.js:16306 THREE.WebGLRenderer: EXT_texture_filter_anisotropic extension not supported.

Is there any way to get the very same version imported from unpkg.com as the examples page is running, or when will it be available?

Thank you,
Richard

**Comment by elalish** [comment, marked as answer]
Indeed, our dev pages run off master, while unpkg only serves releases. You can test our master version by pointing to the one we serve on our dev pages: `https://modelviewer.dev/node_modules/@google/model-viewer/dist/model-viewer.min.js`, though don't use this for production! It could easily be broken by a merge before we've had a full test pass. 

Those extension not supported warnings have been around for some time and shouldn't adversely affect anything. I'm surprised your texture swap is broken; are you sure the error is in model-viewer? Can you post a link that shows the repro?

**Reply by richardrevesz** [reply]
Hi,

thank you for your response!
I was able to load the one you linked and the warnings are gone now (yeah, not errors but warnings).
And you are right the swap is not affected, the issue was on my side.

Best,
Richard
